web/template/payments.gohtml

```diff
@@ -41,10 +41,21 @@
                                     {{ if eq .Source "PHONE" }}
                                         Over the phone
                                     {{ else if eq .Source "MAKE" }}
-                                        Online - Make an LPA
+                                        Online - Make
+                                    {{ else if eq .Source "ONLINE" }}
+                                        Online - Notify
                                     {{ end }}
                                 </dd>
                             </div>
+                            {{ if or (eq .Source "PHONE") (eq .Source "ONLINE") }}
```

Comment:
We don't use it on the Sirius side yet, but when we do we should switch this to use the `locked` property.